test-windows

```diff
@@ -2,10 +2,8 @@
 
 set -euo pipefail
 
-base=cockroach-$(<VERSION).windows-6.2-amd64
+# Download the binary based on the VERSION file. This sets the COCKROACH_VERSION env variable
+# and downloads the binary in ./mnt/
+. "$(dirname "$0")/download_binary.sh" "windows-6.2-amd64.zip"
```

Comment:
I'd either extract `root=$(dirname "$0")` and adjust the invocation of bincheck below to be relative to `$root`—or continue not supporting execution with a different CWD and hardcode this to `./download_binary.sh`.

Reply:
Done, switched to `./download_binary.sh`.